test/ecto_test.exs

```diff
@@ -739,4 +739,145 @@ defmodule Geo.Ecto.Test do
       assert result.coordinates == {42.0, 42.0}
     end
   end
+
+  describe "st_is_collection/1" do
+    test "returns true for a geometry collection" do
+      collection = %Geo.GeometryCollection{
+        geometries: [
+          %Geo.Point{coordinates: {0, 0}, srid: 4326},
+          %Geo.LineString{coordinates: [{0, 0}, {1, 1}], srid: 4326}
+        ],
+        srid: 4326
+      }
+
+      Repo.insert(%LocationMulti{name: "collection", geom: collection})
+
+      query =
+        from(l in LocationMulti,
+          where: l.name == "collection",
+          select: st_is_collection(l.geom)
+        )
+
+      result = Repo.one(query)
+      assert result == true
+    end
+
+    test "returns true for a multi-geometry" do
+      multi_point = %Geo.MultiPoint{
+        coordinates: [{0, 0}, {1, 1}, {2, 2}],
+        srid: 4326
+      }
+
+      Repo.insert(%LocationMulti{name: "multi_point", geom: multi_point})
+
+      query =
+        from(l in LocationMulti,
+          where: l.name == "multi_point",
+          select: st_is_collection(l.geom)
+        )
+
+      result = Repo.one(query)
+      assert result == true
+    end
```

Comment:
@varjas Hrm... it looks like these are failing on `main` after merging with the change from #244 which bumped the PostGIS version. Any ideas? Sample failure: https://github.com/felt/geo_postgis/actions/runs/15375655480/job/43260052513